node/core/approval-voting/src/lib.rs

```diff
@@ -732,6 +751,10 @@ async fn run<B, Context>(
 where
 	B: Backend,
 {
+	if let Err(err) = db_sanity_check(subsystem.db, subsystem.db_config) {
```

Comment:
Should we bail in case we can't read from the db?

Reply:
Yeah, we could so that, it also happens later if not here.